cg-spec/editors_draft.html

```diff
@@ -462,7 +462,53 @@ <h2>Discussion</h2>
     </section>
   </section>
 
-    <section id="n-triples-star">
+  <section id="trig-star">
+    <h2>TriG-star</h2>
+
+    <p>This section describes TriG-star,
+      a minimal extension of the TriG format [[TRIG]]
+      using the same production updates described in <a href="#turtle-star" class="sectionRef"></a>.</p>
+
+    <p class="note">As RDF-star describes <a>embedded triples</a>,
+      which are not necessarily described within any specific graph,
+      either named or default.</p>
+
+    <p>A <dfn>TriG-star document</dfn> defines an <a>RDF-star dataset</a> composed of
+      a single <a>default graph</a>, and zero or more <a>named graphs</a>,
+      all of which are <a>RDF-star graphs</a>.</p>
+
+    <p>The TriG-star grammar contains exactly the same productions updates
+      described in <a href="#turtle-star-grammar" class="sectionRef"></a>.</p>
+
+    <p>TriG-star parsing uses the same updates described in <a href="#turtle-star-parsing" class="sectionRef"></a> as applied to <a data-cite="TRIG#sec-parsing">Section 5 of the TriG specification</a> [[TRIG]].</p>
+
+    <p class="note">As with Turtle-star, the <a href="#grammar-production-embTriple">`embTriple`</a> and annotation <a href="#grammar-production-annotation">`annotation`</a> yield <a>RDF-star triples</a> which are not directly added to `curGraph`.</p>
```

Comment:
I find the mention of `annotation` confusing... First, the annotation syntax *does* produce triples (including the triple that is otherwise embedded as subject). Second, this might give the wrong impression that the annotated triple, on the one hand, and the annotating triples, on the other hand, are produced in *different* graphs (which is not the case!).

I suggest droping the mention to annotation completely
```suggestion
    <p class="note">As with Turtle-star, the <a href="#grammar-production-embTriple">`embTriple`</a> production yields an <a>RDF-star triple</a> which is not directly added to any graph.</p>
```

Reply:
This comes from the Turtle-star Parsing section, and I think is accurate. The `annotation` production doesn't emit a triple, but creates a _curSubject_; it is the sub-productions in the `predicateObjectList` which emit triples using this _curSubject_. Perhaps a note here (and maybe in the Turtle-star section) might improve the intuitive meaning?